indico/modules/categories/controllers/display.py

```diff
@@ -382,22 +387,33 @@ def _process(self):
         return jsonify_data(flash=False, html=html)
 
 
-class RHShowPastEventsInCategory(RHDisplayCategoryBase):
-    """Set whether the past events in a category are automatically displayed or not"""
+class RHShowEventsInCategory(RHDisplayCategoryBase):
```

Comment:
I'd call this `RHShowEventsInCategoryBase`

Reply:
done